### PR TITLE
no more than 16 students

### PR DESCRIPTION
**Pull request by g4s8**
#449 - students limit

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#586](https://github.com/zerocracy/farm/issues/586) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#586](https://github.com/zerocracy/farm/issues/586) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/481319602). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

File: src/main/java/com/zerocracy/pmo/People.java [hunk 1]
```diff
@@ -179,6 +182,19 @@ public void invite(final String uid, final String mentor)
...
+                new Xocument(item.path())
+                    .nodes(
+                        String.format(
+                            "/people/person[mentor/text()='%s']",
+                            mentor
+                        )
+                    ).size() >= People.MAX_STUDENTS
+                ) {
+                throw new SoftException(
+                    "You can not invite more than 16 students"
```
Comment (yegor256) [inline src/main/java/com/zerocracy/pmo/People.java]: @g4s8 let's use `Par` and mention the paragraph  of the policy. See how it's done in all other cases.

**Comment by g4s8** [comment]
@yegor256 fixed, please see new changes

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 Rultor failed to push, please restart merge:
```
To git@github.com:zerocracy/farm.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'git@github.com:zerocracy/farm.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
```

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), total is [+1425](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10169](https://www.0crat.com/u/yegor256)

**Pull request merged**